Runtime/CourseRunner.cs

```diff
@@ -68,24 +74,12 @@ public void Execute()
         /// <summary>
         /// Currently running <see cref="ICourse"/>
         /// </summary>
-        public static ICourse Current
-        {
-            get
-            {
-                return instance == null ? null : instance.course;
-            }
-        }
+        public static ICourse Current => instance == null ? null : instance.course;
 
         /// <summary>
         /// Returns true if the current <see cref="ICourse"/> is running.
         /// </summary>
-        public static bool IsRunning
-        {
-            get
-            {
-                return Current != null && Current.LifeCycle.Stage != Stage.Inactive;
-            }
-        }
+        public static bool IsRunning => Current != null && Current.LifeCycle.Stage != Stage.Inactive;
```

Comment:
What is your opinion on these kind of getter? Checked our code style but they are not mentioned.